package.json

```diff
@@ -10,32 +10,36 @@
     "test": "./node_modules/.bin/truffle test",
     "coverage": "./scripts/coverage-verify.sh 100",
     "compile": "./node_modules/.bin/truffle compile",
+
     "migrateToRedT": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=red-t-identity-admin",
     "initRedT": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=red-t-first-id",
     "deployAnsRedT": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=red-t-identity-admin",
     "updateRedT": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=red-t-identity-admin",
+
     "migrateToRedB": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=red-b-identity-admin",
     "initRedB": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=red-b-first-id",
     "deployAnsRedB": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=red-b-identity-admin",
     "updateRedB": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=red-b-identity-admin",
+
     "migrateLocal": "./node_modules/.bin/truffle migrate --f 1 --to 3 --network=local-admin",
     "initLocal": "./node_modules/.bin/truffle exec ./migrations/4_initialize_contracts.js --network=local-first-id",
     "deployAnsLocal": "./node_modules/.bin/truffle migrate --f 5 --to 5 --network=local-admin",
     "updateLocal": "./node_modules/.bin/truffle migrate --f 6 --to 6 --network=local-admin",
+
     "lint": "./node_modules/.bin/solium -d contracts/",
     "remixd": "./node_modules/.bin/remixd -s $(pwd)/contracts --remix-ide https://remix.ethereum.org",
     "dockerTestnetRun": "docker run --name geth-node -d -p 8545:8545 ethereum/client-go --dev --rpc",
     "dockerTestnetStop": "docker rm geth-node",
     "dockerTestnetLogs": "docker logs geth-node --tail 40 -f",
-    "dockerTestnetConsole": "docker exec -it geth-node geth attach /tmp/geth.ipc"
+    "dockerTestnetConsole": "docker exec -it ethereum-node geth attach /tmp/geth.ipc"
   },
   "author": "Alastria",
   "license": "MIT",
   "dependencies": {
     "@truffle/config": "1.2.33",
-    "@truffle/hdwallet-provider": "1.1.1",
+    "@truffle/hdwallet-provider": "^1.1.1",
     "keythereum": "1.2.0",
     "remixd": "0.2.4-alpha.0",
-    "truffle": "5.1.53"
+    "truffle": "^5.1.53"
```

Comment:
Please, try to fix a version, remove `^`and confirm the version you are using.

Reply:
For example, in your PR I see `truffle 5.3.23`. Confirm the `truffle/hdwallet-provider` version so we can fix it in here.